### PR TITLE
[KARAF-2511] Add karaf-scheduler-example

### PR DESCRIPTION
**Pull request by jbonofre**

### CODE REVIEW

File: examples/karaf-scheduler-example/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Apache Karaf Scheduler example
+
+## Abstract
+
+This example shows you how to create a runnable service periodically exectued by the Apache Karaf scheduler.
```
Comment (fpapon) [inline examples/karaf-scheduler-example/README.md]: fix *executed* word

File: examples/karaf-scheduler-example/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+On a running Karaf instance, register the features repository using:
+
+```
+karaf@root()> feature:repo-add mvn:org.apache.karaf.examples/karaf-scheduler-example-features/4.2.0-SNAPSHOT/xml
+```
+
+Then, you can install the runnable service feature:
+
+```
+karaf@root()> feature:install karaf-scheduler-example-runnable
```
Comment (fpapon) [inline examples/karaf-scheduler-example/README.md]: replace *karaf-scheduler-example-runnable* by *karaf-scheduler-example*

**Review by fpapon — COMMENTED** [review]
Please fix documentation

**Comment by jbonofre** [comment]
Rebased & updated.

**Comment by fpapon** [comment]
LGTM
Thanks !

**Pull request merged**